renpy/common/00splines.rpy

```diff
@@ -62,6 +62,7 @@ init -1500 python:
             self.initialized = None
 
         def init_values(self, sizes):
+            # TODO : check this regarding the new absolute.compute
```

Comment:
This seems to be used in the part where ATL warps between three or more key positions. I think it should definitely support all types of position, and that it's current lack of support for absolutes is a bug in and of itself.
While we're at it, it could be good to support interpolating between positions of different types, as it is not currently supported between only two values, and this code proves it can be done.